lib/firmata.js

```diff
@@ -505,6 +523,12 @@ function Board(port, options, callback) {
     RES_TX3: 0x07,
   };
 
+  this.RESOLUTION = {
+    ADC: null,
```

Comment:
What happens if the user skips the configurationQuery? Do we need defaults values set here instead?